test/rollup_transfer.test.ts

```diff
@@ -23,6 +23,7 @@ function link(contract: any, instance: any) {
 contract("Rollup Transfer Commitment", () => {
```

Comment:
You can use `describe` here to stop truffle-fixtures.js deploying the contracts you don't need.
```suggestion
describe("Rollup Transfer Commitment", () => {
```